test/Microsoft.NET.Publish.Tests/Microsoft.NET.Publish.Tests.csproj

```diff
@@ -32,6 +32,7 @@
     </PackageReference>
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="$(RepositoryRootDirectory)src\Tasks\Microsoft.NET.Build.Tasks\PackageInfoHelpers.cs" Exclude="$(GlobalExclude)" />
```

Comment:
@dsplaisted is it fine to include product code in test ? Or is there any other good way to consume this logic in tests